zephyr/include/sof/trace/preproc.h

```diff
@@ -0,0 +1,23 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2024 Intel Corporation.
+ */
+
+#ifndef __SOF_TRACE_PREPROC_H__
+#define __SOF_TRACE_PREPROC_H__
+
+#ifdef CONFIG_TRACE
+#error "SOF dma-trace (CONFIG_TRACE) not supported in Zephyr builds."
```

Comment:
can we say here ".. use X instead for logs"

Reply:
Added in V2 patch.